### PR TITLE
feat: styling for project cards

### LINKED ISSUE #44767: Improve project blocks on new curriculum superblock pages?
I was looking at the new curriculum, the certification projects blend in with the rest of the projects pretty good. I played around with it in the dev tools real quick and came up with this:
Screen Shot 2022-01-10 at 8 32 22 AM

Wondering if we should do something to distinguish the cert projects from the rest of the blocks. It doesn't have to look like the image.

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #44767

Adds a bit of styling to make the cert project cards stand out.

### CODE REVIEW

**Comment by raisedadead** [comment]
Nice work.

image

Nitpicks: 
- That icon seems to be a bit off from where it should be.
- Overall the title is too long for my attention when I am looking for it. Maybe as a user I will get used to it, but do we want to split it into a title and a subtitle. The card certainly could be larger and doesn't need to follow the same design layout as the others.

**Comment by Sembauke** [comment]
Hi Nich,

Maybe place the Icon on the right side of the screen and make it as big as two title-rows.  As the Icon now appears to be of with the start of the text.  Also it might be handy to fill the icon in, instead of it only having a border. That might be good for attention as well.

Also adding Certification to the title and then explaining it in the subtitle, that it is required for the certification, might be a bit redundant IMO.

**Comment by moT01** [comment]
What about something like this:
Screen Shot 2022-01-14 at 8 09 12 AM

**Comment by nhcarrigan** [comment]
image

What do we think about something like this?

**Comment by raisedadead** [comment]
These are way better @nhcarrigan, I would also go with what @moT01 said about the titles. We should make them terse. 

Also, I think that a larger line height that mates it to the height of the icon would be great. Or maybe it's my OCD triggering, so you can ignore this bit

File: client/src/templates/Introduction/components/block.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.big-block-title > svg {
+  height: 1.5rem;
+  display: inline;
+  width: auto;
+  margin-right: 10px;
+  vertical-align: text-bottom;
```
Comment (ojeytonwilliams) [inline client/src/templates/Introduction/components/block.css:6]:
```suggestion
  height: 1.6rem;
  display: inline;
  width: auto;
  margin-right: 10px;
  vertical-align: bottom;
```
```suggestion
  vertical-align: bottom;
```
bottom:
image

text-bottom:
image

bottom + height: 1.6rem:
image

I think the 3rd's the prettiest.

**Comment by ahmadabdolsaheb** [comment]
Great discussion here. I also updated the original issue with our mockup so it could give us a base on which we could quickly iterate.

It would be helpful to think of "Certification Project" as a tag to categorize cards. 
If the project names are going to be concise, It might make sense to have the tag in the same line as the title. 

If we decide to show an icon, It would be better to display the icon on the right side of the title so the text of the card titles are aligned in the page.

**Comment by ahmadabdolsaheb** [comment]
Wondering how much usage this feature is getting.
Screen Shot 2022-01-22 at 7 43 13 PM

I think the best solutions moving forward is to have a row of tags on the top of the cards

**Comment by ahmadabdolsaheb** [comment]
Here is how it looks now: 

Screen Shot 2022-01-24 at 4 36 59 PM

**Comment by nhcarrigan** [comment]
This is coming together really nicely. 

Currently, clicking on "Survey Form Project" scrolls the block to the top of the page. I don't seem to be able to get to the actual project page from the map. 🤔

**Comment by ahmadabdolsaheb** [comment]
> Currently, clicking on "Survey Form Project" scrolls the block to the top of the page. I don't seem to be able to get to the actual project page from the map. 🤔

It should work properly now.

**Comment by nhcarrigan** [comment]
This is looking much better than what I had. Thanks for all of your help with this @ahmadabdolsaheb

File: client/src/utils/is-new-responsive-web-design-cert.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { SuperBlocks } from '../../../config/certification-settings';
```
Comment (ShaunSHamilton) [inline client/src/utils/is-new-responsive-web-design-cert.ts:1]: My 2 cents: I would prefer these tiny functions going in the `utils/index.ts` file, as opposed to their own.

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

File: client/src/utils/is-new-responsive-web-design-cert.ts (hunk 2 above)
Comment (nhcarrigan) [reply client/src/utils/is-new-responsive-web-design-cert.ts:1]: I'm the opposite tbh - I like having them in a separate file. Easier to find.

**Review by moT01 — APPROVED** [review]
LGTM :tada:  

Looks real good @nhcarrigan @ahmadabdolsaheb

**Pull request merged**